### DISCUSSION TITLE
Thermographic camera

### DISCUSSION THREAD

**Discussion by landermartin** [carla-simulator/carla#6546] [Ideas]
Hello developers
For the development of an autonomous car there are many ways to detect elements on the road. But could a thermographic camera be developed to facilitate the detection of human beings? 
If it is not in the process of development, does anyone know how to attribute temperature to objects? How could this new sensor be developed?
I look forward to hearing from you

**Comment by Kazawaryu** [comment]
Infrared sensors and CMOS technology can be combined to achieve more accurate object detection and classification. This is particularly important for L3 or higher autonomous vehicles, which rely on such information for control and decision-making. While CMOS solutions can provide higher resolution than far infrared sensors for general detection of inanimate objects, only under good lighting and weather conditions. 
At my internship company, we are developing an inspection robot. One function is using infrared sensors to detect potentially dangerous scenarios at substations or factories. The infrared sensor emits a modulated infrared beam, which is received by an infrared receiver, creating a warning line made up of infrared beams. Detect changes in the temperature of inanimate objects and make decisions.
